NeuroID/src/reactNativeLib/java/com/neuroid/tracker/events/NIDTouchEventManager.kt

```diff
@@ -216,4 +239,84 @@ class NIDTouchEventManager(
             lastView = currentView
         }
     }
+
+
+    private fun generateMotionEventValues(motionEvent: MotionEvent): JSONObject {
```

Comment:
were unit tests made for these new methods? if not consider adding in a fast follow PR.

Reply:
Good idea, will follow up in another pr